server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -311,6 +311,7 @@ export class ScribeLambda implements IPartitionLambda {
                                     operation.sequenceNumber,
                                     false,
                                     this.serviceConfiguration.scribe.clearCacheAfterServiceSummary);
+                                this.updateProtocolHead(operation.sequenceNumber);
```

Comment:
As a start, let's delete the ops that client has summarized. Don't update the protocolhead after a service summary.